pkg/s3store/s3store_test.go

```diff
@@ -171,6 +172,58 @@ func TestNewUploadWithMetadataObjectPrefix(t *testing.T) {
 	assert.NotNil(upload)
 }
 
+func TestNewUploadWithSSE(t *testing.T) {
+	mockCtrl := gomock.NewController(t)
+	defer mockCtrl.Finish()
+	assert := assert.New(t)
+
+	sSEPattern := "AES256"
+	s3obj := NewMockS3API(mockCtrl)
+	store := New("bucket", s3obj)
+	store.SSEPattern = sSEPattern
+
+	assert.Equal("bucket", store.Bucket)
+	assert.Equal(s3obj, store.Service)
+	assert.Equal(sSEPattern, store.SSEPattern)
```

Comment:
I'd suggest leaving this one out since you just set it a few lines earlier, and you haven't run any operations on the store since then.